### PR TITLE
Project Title: Team Awesome Book

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
* Setup the required files
* Add functionality to add items to the collection
* Add functionality to remove items on the screen when the ``remove`` button clicked
*  Add functionality to add books to collection and ``local storage``
* Add functionality to remove books from the local storage

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const container = document.querySelector('.container');
+
```
Comment (Ismailco) [inline index.js]: - [ ] You should have a collection that keeps the list of books, kindly create an array to store the books
e.g 
```javascript
const books = [ ]
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// Add book to list
+function addBookToList(book) {
```
Comment (Ismailco) [inline index.js:4]: - [ ] When adding a book to the list, the book persists in the local storage however, when reloading the page, the list of books is not displayed on the page, kindly check how you retrieve data from the Localstorage to fix this issue

Screenshot_20220927_190212

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Ridwanullahi-code, @Osoro254Alex, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Ridwanullahi-code, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**